api.js

```diff
@@ -0,0 +1,99 @@
+const carmi = require('./index')
+const path = require('path')
+const fs = require('fs-extra')
+const {isUpToDate, getDependenciesHashes, analyzeDependencies} = require('./src/analyze-dependencies')
+const getCacheFilePath = require('./src/get-cache-file-path')
+const wrapModule = require('./src/wrap-module')
+const base64ArrayBuffer = require('./bytecode/base64-arraybuffer')
+const {CACHE_SCENARIOS} = require('./src/cache-scenarios')
+const {defaults} = require('lodash')
+
+require('@babel/register')({
```

Comment:
require('@babel/register').revert() can be used for clean up. But in the case of the loader we should call revert after all the files finished transpiling.

Reply:
I'm not sure calling revert will help. The loader may handle one file and then another after a while. Whenever the loader processes a file it needs babel-register to be applied. We can re-apply and cleanup every time the loader processes a file but I'm not sure it will help with performance.